testing/00_title.md

```diff
@@ -44,162 +73,235 @@ plugins: RevealMarkdown, RevealChalkboard, RevealHighlight, RevealMath.KaTeX, Re
 ## Test Types
 
 <ul>
-  <li>Unit test
-  <ul class="fragment fade-up" data-fragment-index="1">
-    <li>Smallest possible unit</li>
+  <li class="fragment fade-up">Exceptions in the code base
```

Comment:
Maybe make Unit test bold to make it clear that we only will focus on them? Perhaps  put them on the top of the list.